src/content/governance/index.md

```diff
@@ -0,0 +1,159 @@
+
+### Community Consensus {#community-consensus}
+
+Unfortunately, there is no clear playbook on how to handle contentious proposals. Since they have no way to force people to adopt network upgrades, Protocol Developers will generally avoid including EIPs that are contentious amongst various stakeholder groups.
+
+As the champion of a contentious EIP, you should try and build consensus, address objections and document community support for your proposal. Given the size and diversity of the Ethereum community, there isn't a single metric (e.g. a coin vote) that can be used, and EIP Champions are expected to adapt to the circumstances of their proposal.
+
+Beyond the security of the Ethereum network, significant weight is usually placed on what Application/Tooling Developers and Application Users value, given that their using and developing on Ethereum is what makes the ecosystem attractive for other stakeholders.
+
+<Divider />
+
+## Handling disagreements {#disagreements}
+
+Having many stakeholders with different motivations and beliefs means that disagreements are not uncommon.
+
+Generally, disagreements are handled with long-form discussion to understand the root of the problem. Typically, one group concedes, or a happy medium is achieved. If one group feels strongly enough, forcing through a particular change could result in a split chain. A split chain is when some stakeholders protest implementing a protocol change resulting in different, incompatible versions of the protocol operating, from which two distinct blockchains emerge.
+
+### The DAO fork {#dao-fork}
+
+Forks are when major technical upgrades or changes need to be made to the network and change the "rules" of the protocol. [Ethereum clients](/developers/docs/nodes-and-clients/) must update their software to implement the new fork rules.
+
+The DAO fork was in response to the [2016 DAO attack](https://www.coindesk.com/understanding-dao-hack-journalists) where an insecure [DAO](/glossary/#dao) contract was drained of over 3.6 million ETH in a hack. The fork moved the funds from the faulty contract to a new contract allowing anyone who lost funds in the hack to recover them.
+
+This course of action was voted on by the Ethereum community. Any ETH holder was able to vote via a transaction on [a voting platform](http://v1.carbonvote.com/). The decision to fork reached over 85% of the votes.
+
+Some miners refused to fork because the DAO incident wasn't a defect in the protocol. They went on to form [Ethereum Classic](https://ethereumclassic.org/).
+
+Watch more on the DAO hack:
+
+<iframe width="100%" height="315px" src="https://www.youtube.com/embed/rNeLuBOVe8A" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+<Divider />
```

Comment:
I think there is significant value in mentioning here that the Ethereum/Ethereum Classif fork is a great example of a healthy fork.  We had two groups who disagreed strongly enough with each other on some core values so as to feel it was worth the loss of network effects (for all of Ethereum) to pursue their specific courses of action.

The fact that people *can* fork is a big part of why Ethereum governance works, and there is great value in making new users of the platform aware of that mechanism.  This example is a pristine example of that in action, and has great educational value that I think we should leverage.

Reply:
Added this section after the DAO fork section:

> ### The utility of forking {#forking-utility}
> 
> The Ethereum/Ethereum Classic fork is an excellent example of a healthy fork. We had two groups who disagreed strongly enough with each other on some core values to feel it was worth the risks involved to pursue their specific courses of action.
>
> The ability to fork in the face of significant political, philosophical or economic differences plays a large part in the success of Ethereum governance. Without the ability to fork the alternative was ongoing in-fighting, forced reluctant participation for those who eventually formed Ethereum Classic and an increasingly differing vision of how success for Ethereum looks.